api/v1beta1/ansibletest_types.go

```diff
@@ -29,73 +29,73 @@ type AnsibleTestSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// +kubebuilder:validation:Optional
 	// Extra configmaps for mounting in the pod.
-	ExtraMounts []extraConfigmapsMounts `json:"extraMounts,omitempty"`
+	ExtraConfigmapsMounts []extraConfigmapsMounts `json:"ExtraConfigmapsMounts,omitempty"`
```

Comment:
Sorry, for not being to explicit. 

```
ExtraConfigmapsMounts []extraConfigmapsMounts `json:"extraConfigmapsMounts,omitempty"`
```

(take a look at the other parameters)

`json:` <- this is name for the parameter in Tempest CR
`ExtraConfigmapsMounts` <- this is name of the parameter in the test-operator code